### PR TITLE
Refine icon spacing

### PR DESCRIPTION
**Pull request by microbit-robert**
Edit icon spacing for new icons.

See #487.

### CODE REVIEW

File: src/documentation/explore/ToolkitIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { Image } from "@chakra-ui/react";
+import { imageUrlBuilder } from "../../common/imageUrlBuilder";
+import { ToolkitImage } from "./model";
+
+interface ToolkitIconProps {
+  icon: ToolkitImage;
+}
+
+const ToolkitIcon = ({ icon }: ToolkitIconProps) => {
+  console.log(icon);
```
Comment (microbit-matt-hillsdon) [inline src/documentation/explore/ToolkitIcon.tsx:15]: Debug

**Comment by microbit-robert** [comment]
I think we should reduce padding-bottom here when all icons are updated:
image

I think we should reduce margin-top and margin-bottom for the topics when all the icons are updated:
image

These changes would look quite odd unless we changed the placeholder icon to match the dimensions of the new icons.

File: src/documentation/explore/ToolkitBreadcrumbHeading.tsx [hunk 2]
```diff
@@ -63,17 +63,8 @@ const ToolkitBreadcrumbHeading = ({
...
-          <Image
-            src={imageUrlBuilder.image(icon.asset).url()}
-            alt=""
-            width="80px"
-            height="64px"
-            borderRadius="lg"
-            mt={1}
-          />
-        )}
+      <HStack align="center" spacing={4}>
```
Comment (microbit-robert) [inline src/documentation/explore/ToolkitBreadcrumbHeading.tsx:66]: I'm a bit unsure on `align="center"` here, but it seems to the yield the best results with inbuilt spacing inside the icons.

File: src/documentation/explore/ToolkitIcon.tsx (hunk 1 above)
Comment (microbit-robert) [reply src/documentation/explore/ToolkitIcon.tsx]: [`5ef1775`](https://github.com/microbit-foundation/python-editor-next/pull/504/commits/5ef1775ee4272c96da9d16ec57f215a26b757b16)

**Comment by microbit-robert** [comment]
> I think we should reduce padding-bottom here when all icons are updated: image
> 
> I think we should reduce margin-top and margin-bottom for the topics when all the icons are updated: image
> 
> These changes would look quite odd unless we changed the placeholder icon to match the dimensions of the new icons.

The spacing here matches the designs, and looks better now that all the icons are in. The padding could be reduced to show more topic entries on a single page.

File: src/documentation/explore/ToolkitBreadcrumbHeading.tsx (hunk 2 above)
Comment (microbit-robert) [reply src/documentation/explore/ToolkitBreadcrumbHeading.tsx:66]: Created a layout issue on smaller screen/sidebar widths are has been resolved with [`e6c13b6`](https://github.com/microbit-foundation/python-editor-next/pull/504/commits/e6c13b656f5a8aa0e08c7c1218a12e678f23a98f).

**Review by microbit-matt-hillsdon — APPROVED** [review]

**Pull request merged**